.github/workflows/build-golang.yml

```diff
@@ -39,35 +50,18 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: 3.9
-      - uses: actions/setup-dotnet@v1
-        with:
-          dotnet-version: '6.0.x'
-      - name: Set up Dotnet (docs)
-        run: |
-          dotnet tool install DefaultDocumentation.Console -g
-        shell: pwsh
       # Set up Okapi-specific Go runtime. Includes setting GOLANG_LD_PATH env var.
       - name: Set up Go (Okapi)
         run: |
           go version
           go install golang.org/x/lint/golint@latest
           go install github.com/jstemmer/go-junit-report@latest
-          go get github.com/posener/goreadme/cmd/goreadme
+          go install github.com/posener/goreadme/cmd/goreadme@latest
           python ../../devops/build_sdks.py --language=golang
         shell: pwsh
         working-directory: go/okapi
         env:
           GO111MODULE: on
-      - name: Build and Test Golang
-        run: |
-          go build      - name: Set up Go (Okapi)
```

Comment:
Don't we need this?  This is setting up Golang, not Dotnet.

Reply:
oof @ my bad paste, I think it should be:
```
- name: Set up Go (Okapi)
        run: |
          go version
          go install golang.org/x/lint/golint@latest
...
```
i.e. remove the first two lines and fix the third.  but I do think we need it